### DISCUSSION TITLE
Set Highlighting Icon to On by Default

### DISCUSSION THREAD

**Discussion by ricraftis** [RyotaUshio/obsidian-pdf-plus#154] [Q&A] [answered]
Hi,

Whenever I open a PDF to annotate, which is generally academic papers, I have to click on the "pen" icon to allow me to highlight. I can't find a switch to set it on by default. Is there one? 

![image](https://github.com/RyotaUshio/obsidian-pdf-plus/assets/121541749/c0495108-b35e-4a76-bce0-4c0a2d215287)

**Comment by RyotaUshio** [comment, marked as answer]
There is an option called **Write highlights to file by default**. Open `obsidian://pdf-plus?setting=defaultWriteFileToggle` from your browser
